helm/hpcc/templates/_helpers.tpl

```diff
@@ -945,23 +949,71 @@ Pass in dict with root and me
 {{- end -}}
 
 {{/*
-A template to generate Sasha service
-Pass in dict me
+A template to generate the type of a service based on the visibility setting
+Pass in dict with .root, .visibility defined
 */}}
-{{- define "hpcc.addSashaService" }}
-{{- $serviceName := printf "sasha-%s" .me.name }}
+{{- define "hpcc.getVisibilityType" }}
```

Comment:
Is this the correct name for the function?  getServiceType()?

Reply:
It is mapping a visibility value to a service type value. I could rename it if you think a different name is less confusing. Though I suspect if I can get boolean returns working I will probably replace it with something like "isVisibilityExternal"